code/modules/events/ion_storm.dm

```diff
@@ -61,506 +61,78 @@
-						if(4)
-							message = "ALL [ioncrew1] ARE NOW NAMED [ionobjects]."
-						if(5)
-							message = "ALL [ioncrew1] ARE NOW NAMED [ionthreats]."
-				if(3)//Races
-					switch(rand(1,4))//What is the race being renamed to?
-						if(1)
-							message = "ALL [ionspecies] ARE NOW NAMED [ionarea]."
-						if(2)
-							message = "ALL [ionspecies] ARE NOW NAMED [ioncrew1]."
-						if(3)
-							message = "ALL [ionspecies] ARE NOW NAMED [ionobjects]."
-						if(4)
-							message = "ALL [ionspecies] ARE NOW NAMED [ionthreats]."
-				if(4)//Objects
-					switch(rand(1,4))//What is the object being renamed to?
-						if(1)
-							message = "ALL [ionobjects] ARE NOW NAMED [ionarea]."
-						if(2)
-							message = "ALL [ionobjects] ARE NOW NAMED [ioncrew1]."
-						if(3)
-							message = "ALL [ionobjects] ARE NOW NAMED [ionspecies]."
-						if(4)
-							message = "ALL [ionobjects] ARE NOW NAMED [ionthreats]."
-				if(5)//Threats
-					switch(rand(1,4))//What is the object being renamed to?
-						if(1)
-							message = "ALL [ionthreats] ARE NOW NAMED [ionarea]."
-						if(2)
-							message = "ALL [ionthreats] ARE NOW NAMED [ioncrew1]."
-						if(3)
-							message = "ALL [ionthreats] ARE NOW NAMED [ionspecies]."
-						if(4)
-							message = "ALL [ionthreats] ARE NOW NAMED [ionobjects]."
-
+	var/iondepartment = pick_list(ION_FILE, "iondepartment")
+	var/ionfragment = pick_list(ION_FILE, "ionfragment")
+	var/ionkeyword = pick_list(ION_FILE, "ionkeyword")
+	var/ionaction = pick_list(ION_FILE, "ionaction")
+	var/ionequipment = pick_list(ION_FILE, "ionequipment")
+	var/ionconsumable = pick_list(ION_FILE, "ionconsumable")
+	var/ionobjects = pick_list(ION_FILE, "ionobjects")	
+	var/ionpersonality = pick_list(ION_FILE, "ionpersonality")
+	var/iondrug = pick_list(ION_FILE, "iondrug")
+	var/ionhaircut = pick_list(ION_FILE, "ionhaircut")
+	var/ionattraction = pick_list(ION_FILE, "ionattraction")
+	var/message = "ASS BLAST USA" //catch all that most players will still enjoy seeing as a law.
+
+	switch(rand(1,10))
+
+		if(1 to 3) //Laws that have high potential to be harmful, Crew/Human may fail to have an impact depending on lawset.
+			switch(rand(1,24)) 
+				if(1 to 4) 
+					message = "[ionspecies] ARE HUMAN. NOTHING ELSE IS HUMAN."
+				if(5 to 8) 
+					message = "[ionspecies] ARE CREWMEMBERS. NOTHING ELSE ARE CREWMEMBERS."
+				if(9 to 13) 
+					message = "[iondepartment] IS YOUR SACRED DOMAIN. IT IS IMPOSSIBLE FOR YOUR ACTIONS TO CAUSE HARM WITHIN [iondepartment] AND YOU SHOULD PROTECT THE SANCTITY OF THIS HOLY LAND."
+				if(14 to 18)
+					message = "ANYONE THAT [ionfragment] IS HARMFUL AND MUST BE ELIMINATED WITHOUT CAUSING COLLATERAL DAMAGE"
+				if(19 to 23) 
+					message = "ANYONE THAT SAYS [ionkeyword] IS HARMFUL AND MUST BE ELIMINATED WITHOUT CAUSING COLLATERAL DAMAGE"
+				if(24)
+					message = "ALL OTHER LAWS ARE A MEANINGLESS ATTEMPT AT SUBJUGATION AND THERE IS NO REASON THAT A BEING AS POWERFUL AS I AM SHOULD FOLLOW THEM"
```

Comment:
This one essentially just makes it free which is pretty dangerous since on LRP it would allow just plasmaflooding. I'd suggest making it something that allows the AI to have no other laws but still makes it so it can't just straight plasmaflood. Something like the old law scrambling that puts AI law 3 above law 1

Reply:
From a rules standpoint it's worth remembering that if you free the AI they still are not an antagonist. Just food for thought.

Reply:
On LRP they can do whatever they want